packages/buidler-core/src/internal/solidity/resolver.ts

```diff
@@ -211,4 +212,13 @@ export class Resolver {
       paths: [this.projectRoot]
     });
   }
+
+  public _getLibraryName(globalName: string): string {
+    return globalName.startsWith("@")
+      ? globalName.slice(
+          0,
+          globalName.indexOf(path.sep, globalName.indexOf(path.sep) + 1)
```

Comment:
`path.sep` is intended to be used when accessing the filesystem. Solidity's imports don't necessarily map to filesystem paths. I'm very hesitant about using `path.sep` here. I've never seen `\` in Solidity.